### ISSUE TITLE
Incorrect TypeScript syntax in codebase

### ISSUE THREAD

**Issue by khacdatdo** [jdalrymple/gitbeaker#3556]
I noticed that there is incorrect TypeScript syntax in the codebase. The syntax is attempting to use "|" to select multiple fields from an interface, which is not valid TypeScript syntax.
[See code here](https://github.com/jdalrymple/gitbeaker/blob/0df30514d4c744489862d57b85bddfde13b4aecb/packages/core/src/resources/Webhooks.ts#L194C1-L195C65)

```typescript
assignees: Pick<UserSchema, 'name|username|avatar_url'>[] | null;
assignee: Pick<UserSchema, 'name|username|avatar_url'> | null;
```

Expected Behavior:
The TypeScript syntax should be corrected to properly select the specified fields from the interface.

Possible Solution:
Correct the TypeScript syntax to use individual field selections with "|".

**Comment by jdalrymple** [comment]
Definitely a typo!

**Comment by jdalrymple** [comment]
:rocket: Issue was released in [`40.0.1`](https://github.com/jdalrymple/gitbeaker/releases/tag/40.0.1) :rocket:
